pkg/components/master.go

```diff
@@ -159,6 +159,7 @@ func (m *master) createInitScript() string {
 
 	script := []string{
 		initJobWithNativeDriverPrologue(),
+		"/usr/bin/yt execute master_exit_read_only '{}'",
```

Comment:
Wouldn't it break working with old images, if this command fails with "command maxter_exit_read_only not found"? Should we add "|| true" here?